src/pug/includes/head.pug

```diff
@@ -10,8 +10,10 @@ meta(name="description" content="swag opportunities for developers")
 link(rel="icon" type="image/png" href="/assets/img/logo.png")
 link(rel="apple-touch-icon" size="128x128" href="/assets/img/logo.png")
 
-link(rel="stylesheet" href=bustedAssets.css)
-link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
+style
+	!=inlinedAssets.get('css/index.css')
+
+link(rel="stylesheet" data-href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
```

Comment:
I think this still needs a noscript version

Reply:
`Selectr` is used in buttons section of page. We hide that section if JS is disabled. So IMO loading CSS file in noscript version doesn't make sense.

Please let me know if I missed anything.

Reply:
Aah, you're right